server/integration-tests/fast.sh

```diff
@@ -2,9 +2,13 @@
 
 set -e
 
+# Switch Java to v8 in runtime from current context
+. switch_java_version.sh
+
 ./compile_calabash.sh
 ./compile_tests.sh
 ./build_test_apk_fast.sh
+./setup_unit_tests_fast.sh
 
-cd calabash-test-suite-fast
```

Comment:
Subshell and call `bundle install`

```
(
  cd calabash-test-suite-fast
  bundle install
  bundle exec calabash-android run unittest.apk
)
```

Reply:
`bundle install` executes on previous step `setup_unit_tests_fast.sh`, because _setup_unit_tests_fast_ use `bundle exec calabash-android resign unittest.apk`